### ISSUE TITLE
Add flag to disable multi line mode, and disable by default

### ISSUE THREAD

**Issue by AndrewRayCode** [svermeulen/vim-NotableFt#3]
It's jarring to hit `fi` and then lose my cursor as it jumps down 20 lines unexpectedly. If there's no match on the current line, by default, the cursor shouldn't be moved. I think that multiline search would make more sense as an opt-in behavior.

**Comment by svermeulen** [comment]
Yep, this could be made optional.  Maybe even off by default, as you say, since that is vim defaults.

It's also worth noting that unlike `/` search, the multi-line search won't continue past the end of the screen, until you scroll down manually.
